### PR TITLE
Update 'README.md'

### PR DESCRIPTION
**Pull request by tomkirbygreen**
Update `README.md` for .NET 6, Unity and 'legacy' URLs of the form `ably.io`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,17 +52,13 @@ Implementation note for Unity. The library creates a number of threads and all c
...
-`.NET 6` and `Visual Studio 2022`, both currently in preview, at not supported at this time but will be supported when they are officially released.
-
 ### Portable Class Libraries
 
 A Portable Class Library version is not available. Portable Class Libraries are now considered deprecated and shared code should be packaged in .NET Standard Libraries. More information can be found [here](https://docs.microsoft.com/en-us/xamarin/cross-platform/app-fundamentals/code-sharing).
 
 ## Push notification
 
-The Ably.net library fully supports Ably's push notifications. The feature set consists of two distinct areas: [Push Admin](https://ably.com/documentation/general/push/admin), [Device Push Notifications](https://ably.com/documentation/realtime/push). 
+The `ably_dotnet` library fully supports Ably's push notifications. The feature set consists of two distinct areas: [Push Admin](https://ably.com/documentation/general/push/admin), [Device Push Notifications](https://ably.com/documentation/realtime/push).
```
Comment (owenpearson) [inline README.md:61]: Is the underscore here meant to be a hyphen? I can't find any references to `ably_dotnet` anywhere by way of Google.

File: README.md [hunk 2]
```diff
@@ -411,7 +407,7 @@ DateTimeOffset time = await client.TimeAsync();
 
 ### Increase Transport send and receive buffers
 
-In .NET Framework projects, we discovered issues with the .NET implementation of the web socket protocol during times of high load with large payloads (over 50kb). This is better described in https://github.com/ably/ably-dotnet/issues/446
+In .NET Framework projects, we discovered issues with the .NET implementation of the websocket protocol during times of high load with large payloads (over 50kb). This is better described in https://github.com/ably/ably-dotnet/issues/446
```
Comment (owenpearson) [inline README.md:410]: Consider 'WebSocket' (this is how it's stylised in the original RFC and on wikipedia)

File: README.md [hunk 3]
```diff
@@ -498,7 +494,7 @@ You can also view the [community reported GitHub issues](https://github.com/ably
 
 ## Building and Packaging
 
-The build scripts are written using ```fake``` and need to be run on Windows with Visual Studio 2017 or Visual Studio 2019 installed. Fake and nuget.exe can be installed via [chocolatey](https://chocolatey.org)
+The build scripts are written using ```fake``` and need to be run on Windows with Visual Studio 2017 or Visual Studio 2019 or Visual Studio 2022 installed. Fake and nuget.exe can be installed via [chocolatey](https://chocolatey.org)
```
Comment (owenpearson) [inline README.md:497]: This sentence reads a bit odd now, consider:
- "and need to be run on Windows with Visual Studio 2017, 2019, or 2022"
- "and need to be run on Windows with Visual Studio (versions 2017, 2019, and 2022 are supported)"
- "and need to be run on Windows with Visual Studio (versions 2017 upwards are supported)"

**Review by owenpearson — CHANGES_REQUESTED** [review]
LGTM, just one possible typo and a couple of minor suggestions.

**Comment by tomkirbygreen** [comment]
Closing this as we're taking a slightly different route to .NET 6 by way of removing Windows 7 as a supported platform first.

**Pull request closed without merging**